.github/workflows/release-vm-console-proxy.yaml

```diff
@@ -0,0 +1,40 @@
+name: Release VM Console Proxy
+
+on:
+  repository_dispatch:
+    types: [release-vm-console-proxy]
+
+jobs:
+  release-vm-console-proxy:
+    name: Release VM Console Proxy
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          release_version: ${{ github.event.client_payload.release_version }}
+      - run: |
+          RELEASE_VERSION=${{ github.event.client_payload.release_version }}
+          OUTPUT_FILE=./data/vm-console-proxy-bundle/vm-console-proxy-${RELEASE_VERSION}.yaml
+          mkdir -p ./data/vm-console-proxy-bundle
+          curl -L https://github.com/kubevirt/vm-console-proxy/releases/download/${RELEASE_VERSION}/vm-console-proxy.yaml > ${OUTPUT_FILE}
+
+      - name: Create pull request
+        uses: peter-evans/create-pull-request@v4
+        with:
+          release_version: ${{ github.event.client_payload.release_version }}
+          token: ${{ secrets.GH_ACCESS_TOKEN }}
+          commit-message: "chore: update kubernetes manifests to ${{ github.event.client_payload.release_version }}"
+          committer: Ben Oukhanov <boukhanov@redhat.com>
+          author: Ben Oukhanov <boukhanov@redhat.com>
```

Comment:
can you use some bot name and nonexisting email? Like `vm-console-proxy update`.

Reply:
Done, we're using GitHub Action bot.